charts/flink-job/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "1.0"
 description: Flink job cluster on k8s
 name: flink-job
-version: 0.1.1
+version: 0.1.2
```

Comment:
Always set a default in the `values.yaml` file and add an appropriate documentation string.

Reply:
sure, i just couldn't find the default value so didn't want to cause issue when others update the chart.